content/posts/looker-pdt-to-dbt-part-one.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Converting Looker PDTs to dbt models: Reasons why (and why not)"
+date: "2018-12-25"
+author: "Dylan Baker"
+---
+
+Over the past few years, Looker has become a market-leading BI tool. Its data governance and self-service capabilities have now become paramount to thousands of BI stacks. For many, the Third Wave of BI (TM Frank Bien) is well and truly here.
```

Comment:
I assume you're going to change the `(TM Frank Bien)` part?

Reply:
I wasn't... But I guess what you're saying is I should?

Reply:
No. It's not. He just says it in every talk he gives. It was meant as a joke. Clearly hasn't landed though.